From 816095fe1f31bd3b9cd51ef22b77c85c47dabc91 Mon Sep 17 00:00:00 2001
From: Palmer Cox <p@lmercox.com>
Date: Mon, 3 Aug 2026 01:01:19 +0000
Subject: [PATCH 3/3] Increase the timeout to 100 minutes up from 10

We need the longer timeout since a few tests are failing since they
can't complete in 10 minutes on the sifive-u74.
---
 run.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/run.sh b/run.sh
index 3cda41b7561..dc2a32a4895 100755
--- a/run.sh
+++ b/run.sh
@@ -10,7 +10,7 @@ source util/setchplenv.bash
 export MACHINE_TYPE="$(uname -m)"
 
 export CHPL_TEST_PERF_DIR="$CHPL_HOME/test/perfdat/clbg_comparison"
-export CHPL_TEST_TIMEOUT=600
+export CHPL_TEST_TIMEOUT=6000
 
 start_test --performance --numtrials 10 test/studies/shootout/submitted
 start_test --performance --numtrials 10 test/release/examples/benchmarks/shootout/chameneosredux.chpl
-- 
2.54.0

